anchore/syft
# update to point at new script for new release

Small patch to use new upstream script which adds `all` as a valid case for downloading goreleaser.

We should eventually migrate to one of the official install instructions here:
https://goreleaser.com/install/

To test:

`make bootstrap` <-- this should now return status code 0

Important changes:
https://gist.github.com/spiffcs/479255e52143bb68bb0b2974080bb0a4#file-goreleaser-sh-L232-L253

Add `all` to allow the script to consume the new release binary.

Signed-off-by: Christopher Angelo Phillips <christopher.phillips@anchore.com>
